packages/@webex/webex-sign-in-page/package.json

```diff
@@ -7,18 +7,11 @@
   "keywords": [],
   "author": "devsupport@webex.com",
   "license": "MIT",
-  "dependencies": {
```

Comment:
Why did we need these changes?

Reply:
These dependencies should never be here. But anyways in the next PR all the dependencies will be added along with the scripts